arches/app/models/models.py

```diff
@@ -768,6 +787,27 @@ class Meta:
             ("no_access_to_nodegroup", "No Access"),
         )
 
+    @classmethod
+    def check(cls, **kwargs):
+        errors = super().check(**kwargs)
+        if kwargs["databases"]:
+            errors.extend(cls._check_root_node())
+        return errors
+
+    @classmethod
+    def _check_root_node(cls):
```

Comment:
This check ends up being run prior to migrations which throws an error because rootnode is not yet available.

```django.db.utils.ProgrammingError: column node_groups.rootnodeid does not exist
LINE 1: ..."cardinality", "node_groups"."parentnodegroupid", "node_grou...
```

Reply:
Yeah, I don't like telling people to `--skip-checks` with every migrate call, so I'm going to back out that release note advice and look into skipping this on the migrate command.

Reply:
I'll just remove this and add it back if I get a positive response here before Django 5.2:
🤞 https://code.djangoproject.com/ticket/35920